src/requestLogging/requestLogging.ts

```diff
@@ -160,3 +167,28 @@ export const createMiddleware = <StateT extends State, CustomT>(
       throw err;
     }
   };
+
+/*
+ * Creates a logger context storage instance
```

Comment:
You mentioned earlier that you didn't want the `AsyncLocalStorage` to be global; is there any caution that we should provide here around appropriately scoping the storage?

I wonder if we should also mention the performance penalty async hooks may incur, and recommend a Node.js version (16.2 or newer?) that is appropriately optimised.

Reply:
oh as in initially I was literally just exporting it like this. So there could and only would have been only one loggerContext instance which would always be exported even if they didn't use it aha

`export const loggerContext = new AsyncLocalStorage<Fields>();`